stable/insights-agent/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: goldilocks
+  repository: https://charts.fairwinds.com/incubator
```

Comment:
Once #145 gets merged, you will want to change this. My guess is it will merge before this one.

Reply:
Done